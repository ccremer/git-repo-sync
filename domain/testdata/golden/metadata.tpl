{{`{{ .Metadata.Repository.Namespace }}`}} = {{ .Metadata.Repository.Namespace }}
{{`{{ .Metadata.Repository.CommitBranch }}`}} = {{ .Metadata.Repository.CommitBranch }}
{{`{{ .Metadata.Repository.DefaultBranch }}`}} = {{ .Metadata.Repository.DefaultBranch }}
{{`{{ .Metadata.Repository.RootDir }}`}} = {{ .Metadata.Repository.RootDir }}
{{`{{ .Metadata.Repository.FullName }}`}} = {{ .Metadata.Repository.FullName }}
{{`{{ .Metadata.Repository.Name }}`}} = {{ .Metadata.Repository.Name }}
{{`{{ .Metadata.Template.RelativePath }}`}} = {{ .Metadata.Template.RelativePath }}
{{`{{ .Metadata.Template.Permissions }}`}} = {{ .Metadata.Template.Permissions }}
