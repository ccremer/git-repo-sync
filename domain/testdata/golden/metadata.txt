{{ .Metadata.Repository.DefaultBranch }} = master
{{ .Metadata.Repository.RootDir }} = repos/github.com/ccremer/greposync
{{ .Metadata.Repository.FullName }} = github.com/ccremer/greposync
{{ .Metadata.Repository.Name }} = greposync
{{ .Metadata.Repository.Namespace }} = ccremer
{{ .Metadata.Repository.CommitBranch }} = greposync-update
{{ .Metadata.Template.RelativePath }} = testdata/golden/metadata.tpl
{{ .Metadata.Template.Permissions }} = 0644
